driver.md

```diff
@@ -115,7 +139,7 @@ To do so, we'll extend sort `JSON` with some EVM specific syntax, and provide a
            <nonce> NONCE </nonce>
            ...
          </account>
-         <touchedAccounts> _ => SetItem(MINER) </touchedAccounts>
+         <touchedAccounts> _ => keys(#parseAccessList(TA)) SetItem(MINER) </touchedAccounts>
```

Comment:
Make sure you merge master (which handles the `touchedAccounts` cell differently). Then you can put `<touchedAccounts>` in the `<substate>` cell so that it's tracked properly as a transaction-wide context.